membership_prorrate/__manifest__.py

```diff
@@ -4,18 +4,18 @@
 
 {
     'name': 'Prorrate membership fee',
-    'version': '8.0.1.0.0',
+    'version': '10.0.1.0.0',
     'license': 'AGPL-3',
     'category': 'Association',
     'author': 'Antiun Ingeniería S.L., '
```

Comment:
Remove `Antiun` and `Serv. Tecnol... Baeza` and put only the new company `Tecnativa`